<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Création d'une partie</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblGameName">
          <property name="text">
           <string>Nom de la partie : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtGameName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblMaxPlayer">
          <property name="text">
           <string>Nombre max de joueurs : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbxNbMaxPlayers"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblGameType">
          <property name="text">
           <string>Type de jeu : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxGameType"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblGameMap">
          <property name="text">
           <string>Carte : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxGameMap"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblGamePassword">
          <property name="text">
           <string>Mot de passe(*) : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtGamePassword"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCreateGame">
          <property name="text">
           <string>Créer la partie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
